.travis.yml

```diff
@@ -35,19 +35,41 @@ install: bash ./.travis/travis-dependencies.sh
 
 script: bash ./.travis/travis-compile.sh
 
+
+# Builds for pull requests skip the deployment step altogether
 deploy:
-  provider: releases
-  api_key:
-    secure: "mLMF41q7xgOR1sjczsilEy7HQis2PkZCzhfOGbn/8FoOQnmmPOZjrsdhn06ZSl3SFsbfCLuClDYXAbFscQmdgjcGN5AmHV+JYfW650QEuQa/f4/lQFsVRtEqUA1O3FQ0OuRxdpCfJubZBdFVH8SbZ93GLC5zXJbkWQNq+xCX1fU="
-  file_glob: true
-  file: "build/Cockatrice-*"
-  prerelease: true
-  skip_cleanup: true
-  overwrite: true
-  on:
-    tags: true
-    repo: Cockatrice/Cockatrice
-    condition: $BUILDTYPE = Release
+# Deploy configuration for "beta" releases
+  - provider: releases
+    api_key:
+      secure: "mLMF41q7xgOR1sjczsilEy7HQis2PkZCzhfOGbn/8FoOQnmmPOZjrsdhn06ZSl3SFsbfCLuClDYXAbFscQmdgjcGN5AmHV+JYfW650QEuQa/f4/lQFsVRtEqUA1O3FQ0OuRxdpCfJubZBdFVH8SbZ93GLC5zXJbkWQNq+xCX1fU="
+    skip_cleanup: true
+    name: "Cockatrice $TRAVIS_TAG"
+    body: "Beta release of Cockatrice"
+    file_glob: true
+    file: "build/Cockatrice-*"
+    overwrite: true
+    draft: false
+    prerelease: true
+    on:
+      tags: true
+      repo: Cockatrice/Cockatrice
+      condition: "$BUILDTYPE = Release && $TRAVIS_TAG =~ ([0-9]|[1-9][0-9])(.([0-9]|[1-9][0-9])){2}-beta(|.([2-9]|[1-9][0-9]))$"
```

Comment:
You say this condition twice. Is that correct?

Reply:
What do you mean @ZeldaZach ?

- travis.yml
    - **beta config:** only on tags & on main repo & release build (not debug!) & **beta regex**
    - **stable: config:** only on tags & on main repo & release build (not debug!) & **stable regex**
- appveyor.yml
    - **beta config:** only on tags & **beta regex**
    - **stable: config:** only on tags & **stable regex**

The basic condition package always checks for general deploy rules, the same we had before.
But the deploy provider within each config file differentiate on the regex.